save-frontend/src/main/kotlin/org/cqfn/save/frontend/components/TopBar.kt

```diff
@@ -206,3 +212,7 @@ fun topBar() = fc<TopBarProps> { props ->
         attrs.isOpen = isLogoutModalOpen
     }
 }
+
+private fun userSettings(userName: String) {
```

Comment:
Extremely confusing method naming, it should at least mention that it will cause redirect to a different page

Reply:
Remove method